src/openrct2-ui/windows/ServerList.cpp

```diff
@@ -267,22 +268,17 @@ class ServerListWindow final : public Window
         if (text.empty())
             return;
 
-        std::string temp = static_cast<std::string>(text);
+        u8string temp = static_cast<u8string>(text);
```

Comment:
The cast already states the type explicitly, so no need to state this twice.

```suggestion
        auto temp = static_cast<u8string>(text);
```

Reply:
[6f4df6e](https://github.com/OpenRCT2/OpenRCT2/pull/20004/commits/6f4df6e38ba05451cee9494f4e70063d5d5a10cc)